Update RunAsUsername to beta (#17460)
kubernetes/website: content/en/docs/tasks/configure-pod-container/configure-runasusername.md

{{% capture overview %}}

{{< feature-state for_k8s_version="v1.17" state="beta" >}}

This page shows how to enable and use the `RunAsUserName` feature for pods and containers that will run on Windows nodes. This feature is meant to be the Windows equivalent of the Linux-specific `runAsUser` feature, allowing users to run the container entrypoints with a different username that their default ones.

{{< note >}}

> Currently this feature is in beta state. The overall functionality of the feature will not change, but there may be some changes regarding the username validation. Please take this into consideration when testing or adopting this feature.

Reviewer: ```suggestion
This feature is in beta. The overall functionality for `RunAsUserName` will not change, but there may be some changes regarding the username validation.
```
Reworded as the feature gate is now enabled by default.

Author: Thanks!

> ### Enable the WindowsRunAsUserName feature gate

Reviewer: Does this heading still make sense?

Author: I guess it's still there if you switch the view to 1.16, so no. Removing
